Makefile

```diff
@@ -477,21 +477,21 @@ proto-update-deps:
 
 # Run a 4-node testnet locally
 build-docker-local-ethermint:
-	@$(MAKE) -C networks/local
+	@$(MAKE) docker-localnet
 
 # Run a 4-node testnet locally
 localnet-start: localnet-stop
 ifeq ($(OS),Windows_NT)
-	mkdir build &
+	mkdir localnet-setup &
 	@$(MAKE) docker-localnet
 
-	IF not exist "build/node0/$(ETHERMINT_BINARY)/config/genesis.json" docker run --rm -v $(CURDIR)/build\ethermint\Z ethermintd/node "ethermintd testnet --v 4 -o /ethermint --keyring-backend=test"
+	IF not exist "build/node0/$(ETHERMINT_BINARY)/config/genesis.json" docker run --rm -v $(CURDIR)/build\ethermint\Z ethermintd/node "./ethermintd testnet --v 4 -o /ethermint --keyring-backend=test --ip-addresses 192.168.10.2,192.168.10.3,192.168.10.4,192.168.10.5"
 	docker-compose up -d
 else
-	mkdir -p ./build/
+	mkdir -p localnet-setup
 	@$(MAKE) docker-localnet
 
-	if ! [ -f build/node0/$(ETHERMINT_BINARY)/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/ethermint:Z ethermintd/node "ethermintd testnet --v 4 -o /ethermint --keyring-backend=test"; fi
+	if ! [ -f localnet-setup/node0/$(ETHERMINT_BINARY)/config/genesis.json ]; then docker run --rm -v $(CURDIR)/localnet-setup:/ethermint:Z ethermintd/node "./ethermintd testnet --v 4 -o /ethermint --keyring-backend=test --ip-addresses 192.168.10.2,192.168.10.3,192.168.10.4,192.168.10.5"; fi
```

Comment:
this is the output I got after running `make localnet-start`

First time:
```
if ! [ -f localnet-setup/node0/ethermintd/config/genesis.json ]; then docker run --rm -v /Users/akashkhosla/go/src/github.com/tharsis/ethermint/localnet-setup:/ethermint:Z ethermintd/node "./ethermintd testnet --v 4 -o /ethermint --keyring-backend=test --ip-addresses 192.168.10.2,192.168.10.3,192.168.10.4,192.168.10.5"; fi
Successfully initialized 4 node directories
docker-compose up -d
Docker Compose is now in the Docker CLI, try `docker compose up`

Creating network "ethermint_localnet" with driver "bridge"
ERROR: Pool overlaps with other one on this address space
make: *** [localnet-start] Error 1
```

Second time:
```
if ! [ -f localnet-setup/node0/ethermintd/config/genesis.json ]; then docker run --rm -v /Users/akashkhosla/go/src/github.com/tharsis/ethermint/localnet-setup:/ethermint:Z ethermintd/node "./ethermintd testnet --v 4 -o /ethermint --keyring-backend=test --ip-addresses 192.168.10.2,192.168.10.3,192.168.10.4,192.168.10.5"; fi
docker-compose up -d
Docker Compose is now in the Docker CLI, try `docker compose up`

Creating network "ethermint_localnet" with driver "bridge"
ERROR: Pool overlaps with other one on this address space
make: *** [localnet-start] Error 1
```

Reply:
@khoslaventures thank you! Fixed with a6fa95f0fb42692588581829c55438c13934383d.